app/views/state_file/questions/az_qualifying_organization_contributions/_form.html.erb

```diff
@@ -1,7 +1,7 @@
-<% @main_heading = t('.main_heading', filing_year: current_tax_year) %>
+<% @main_heading_no_html = strip_tags(t('.main_heading_html')) %>
 <% show_yes_no = @contribution_count == 0  %>
 
-<% content_for :page_title, @main_heading %>
+<% content_for :page_title, @main_heading_no_html %>
```

Comment:
How does this look in the browser tab?

Reply:
like what does it look like without the html or with it?